src/componentsNext/FilePicker/FilepickerItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, Fragment } from "react";
+import {
+  FilepickerClose,
+  FilepickerListItem,
+  FilepickerSeparator,
+} from "~/design/Filepicker";
+
+import { NodeSchemaType } from "~/api/tree/schema/node";
+import { fileTypeToIcon } from "~/api/tree/utils";
+import { twMergeClsx } from "~/util/helpers";
+
+export type FilepickerItemType = {
```

Comment:
For a prop type I think it might be a bit better to end call it `FilepickerItemProps` instead of `FilepickerItemType` (or after the suggested name change below, even `FileListProps`). 

*_I sometimes also end a type name with ...Type, but mostly on a type that I infer from a schema._